tests/integration/reqs/test_feature.py

```diff
@@ -0,0 +1,29 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.it_utils import test_async_and_sync
+from xrpl.models.requests import Feature
+
+AMENDMENT = "8CC0774A3BF66D1D22E76BBDA8E8A232E6B6313834301B3B23E8601196AE6455"
+
+
+class TestAMMInfo(IntegrationTestCase):
+    @test_async_and_sync(globals())
+    async def test_basic_functionality(self, client):
+        response = await client.request(Feature())
+        features = response.result["features"]
+
```

Comment:
```suggestion
self.assertEqual(len(features), 71)
```

This type of a "static-check" could help xrpl-py stay in-sync with new amendments (or) fixes to rippled

Reply:
This test shouldn't have to be updated every time we want to update rippled. There can be other processes for that.

Reply:
alright